src/pal/src/exception/seh-unwind.cpp

```diff
@@ -491,7 +491,17 @@ BOOL PAL_VirtualUnwindOutOfProc(CONTEXT *context,
 
     LibunwindCallbacksInfo.Context = context;
     LibunwindCallbacksInfo.readMemCallback = readMemCallback;
+#if UNWIND_CONTEXT_IS_UCONTEXT_T
     WinContextToUnwindContext(context, &unwContext);
+#else
+    st = unw_getcontext(&unwContext);
```

Comment:
@kangaroo  Ah, right. Ok, then passing NULL as you've suggested before sounds like the right way to go.